lib/Twig/Environment.php

```diff
@@ -641,7 +641,7 @@ public function compileSource($source, $name = null)
             $compiled = $this->compile($this->parse($this->tokenize($source, $name)), $source);
 
             if (isset($source[0])) {
-                $compiled .= '/* '.str_replace(array("\r\n", "\r", "\n"), "*/\n/* ", str_replace('*/', '*//*', $source))."*/\n";
+                $compiled .= '/* '.strtr($source, array('*/' => '*//*', "\r\n" => "*/\n/* ", "\r" => "*/\n/* ", "\n" => "*/\n/* "))."*/\n";
```

Comment:
because generated templates are LF-only files and also because twig already makes no difference and replaces CRLF/CR by LF
at least that's my reasoning :)